hedera-node/test-clients/build.gradle.kts

```diff
@@ -62,9 +62,19 @@ val ciCheckTagExpressions =
         "hapiTestMisc" to
             "!(CRYPTO|TOKEN|SMART_CONTRACT|LONG_RUNNING|RESTART|ND_RECONNECT|EMBEDDED|UPGRADE)"
     )
+val prCheckInitialPorts =
+    mapOf(
+        "hapiTestCrypto" to "30000",
+        "hapiTestToken" to "31000",
+        "hapiTestRestart" to "32000",
+        "hapiTestSmartContract" to "33000",
+        "hapiTestNDReconnect" to "34000",
+        "hapiTestTimeConsuming" to "35000",
+        "hapiTestMisc" to "36000"
+    )
```

Comment:
Is this hardcoding each PR check to use different ports?

Reply:
Yes I can't explain why it can matter since we'd imagine each check already running in isolation...but this seems to reduce port `BindException`'s. 🤷

Reply:
Ports can be bound for a lot of reasons in a K8S container, including by other containers on the same host in some setups.
The safest option for ensuring ports are available is to tell Kubernetes, in the pod definition, that the software within the container uses that port.  Kubernetes will then ensure the container has that port available on its localhost address (which may be shared with other containers on the same host) and bind it to the external network interfaces, possibly shifted via NAT, if requested.

Example:
```yaml
    spec:
      containers:
      - name: HapiTestReconnect
        image: hapi-test-ci-image
        ports:
        - containerPort: 34000
```

We probably need to work with devops to ensure our CI definitions include correct container port declarations.

Reply:
I suspect you are right @jsync-swirlds 💯